### ISSUE TITLE
Loading a sample doesn't check if the api_set is supported

### ISSUE THREAD

**Issue by AlexJerabek** [OfficeDev/script-lab#953]
### Bug Report

- **Host**: N/A
- **OS**: N/A
- **Browser**: N/A
- **Environment**: prod

#### Expected behavior:

Loading a sample marked as a higher requirement set than supported (using the `api_set` key) should fail.

#### Actual behavior:

Samples with unsupported features are successfully loaded. It should only be noted that the call to `isSetSupported` in Script Lab is using the deprecated variant that takes in a number instead of a string. This means won't behave correctly with requirement sets like ExcelApi 1.10. There's also no consideration for online-only requirement sets, like `ExcelOnlineApi 1.1`.

#### Steps to Reproduce:

1. Load the "Track changes" sample in Word on Windows (it uses online-only API and is marked as 1.4; Word on Windows only supports up to 1.3)

#### Screenshot:
What happens:
![a055d04d-fab4-428f-9721-efa55df94f97](https://user-images.githubusercontent.com/38896772/177407826-83fd71ed-0d5c-48c0-895f-45ddc164a3b5.png)

What should happen:
![8b12b77d-05ad-4bf9-afde-9eab2b0c2322](https://user-images.githubusercontent.com/38896772/177407834-de802902-c41e-46b6-83b4-6aac19217d06.png)

**Comment by chenxizhang** [comment]
Good point. Per my understanding, it's better to always check the API set in our code...

**Comment by ElizabethSamuel-MSFT** [comment]
Closing all issues since this repo is being archived and no longer maintained.
